var/spack/repos/builtin/packages/r-edger/package.py

```diff
@@ -37,8 +37,10 @@ class REdger(RPackage):
     homepage = "https://bioconductor.org/packages/edgeR/"
     git      = "https://git.bioconductor.org/packages/edgeR.git"
 
+    version('3.22.3', commit='e82e54afc9398ac54dc4caba0f7ae5c43e572203')
     version('3.18.1', commit='101106f3fdd9e2c45d4a670c88f64c12e97a0495')
 
     depends_on('r-limma', type=('build', 'run'))
     depends_on('r-locfit', type=('build', 'run'))
+    depends_on('r@3.5.0:3.5.9', when='@3.22.3')
     depends_on('r@3.4.0:3.4.9', when='@3.18.1')
```

Comment:
Any idea what range of versions require which range of R versions?

Reply:
bioconductor packages are version locked to a specific bioconductor release & r version, so 3.22.3 is part of bioconductor 3.7, which depends on R 3.5 (as far as I understand it):
https://www.bioconductor.org/about/release-announcements/